crates/bevy_app/src/lib.rs

```diff
@@ -25,9 +25,11 @@ pub mod prelude {
         app::App,
         main_schedule::{
             First, FixedFirst, FixedLast, FixedPostUpdate, FixedPreUpdate, FixedUpdate, Last, Main,
-            PostStartup, PostUpdate, PreStartup, PreUpdate, SpawnScene, Startup, StateTransition,
-            Update,
+            PostStartup, PostUpdate, PreStartup, PreUpdate, SpawnScene, Startup, Update,
         },
         DynamicPlugin, Plugin, PluginGroup,
     };
+
+    #[doc(hidden)]
```

Comment:
Why does this need to be doc(hidden)? Why does this import need to be `pub`?

Reply:
I added the #[doc(hidden)] to match the crate above, but in theory this can just be removed here, and kept in the `bevy_ecs` prelude.

My main rational for having it in this prelude was since most of the other pre-defined schedules are in `bevy_app`, as was the schedule this replaces.

Reply:
IMO we should avoid complicating this: just take it out of the `bevy_app` prelude for now.